connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainer.kt

```diff
@@ -407,6 +425,22 @@ internal class StripeConnectWebViewContainerImpl<Listener, Props>(
         }
     }
 
+    private inline fun <reified T> tryDeserializeWebMessage(
+        webFunctionName: String,
+        message: String,
+    ): T? {
+        return try {
+            ConnectJson.decodeFromString<T>(message)
+        } catch (e: IllegalArgumentException) {
+            controller?.onErrorDeserializingWebMessage(
+                webMessage = message,
+                error = "Unable to deserialize message from $webFunctionName",
```

Comment:
This seems redundant. The event name is already `component.web.error.deserialize_message`. We should replace `error` with `function` and value `$webFunctionName`, similar to event  `component.web.warn.unrecognized_setter_function`.